### PR TITLE
Delete eventing ns from manifests

### PR DESCRIPTION
**Pull request by pierDipi**
Same as https://github.com/openshift/knative-eventing/pull/1877 but for SO due to https://github.com/openshift/knative-eventing/pull/1877#discussion_r968329524

### CODE REVIEW

File: vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go [hunk 1]
```diff
@@ -104,6 +104,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ke *v1alpha1.KnativeEven
 		return err
 	}
 	stages := common.Stages{
+		common.FilterNamespace("knative-eventing"),
```
Comment (matzew) [inline vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go:107]: So we exclude the namespace kind with the name.

Is patching vendored code really better, than manfiests?

Comment (pierDipi) [reply vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go:107]: Yes, because manifests are in a versioned path directory

Comment (pierDipi) [reply vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go:107]: which forces us to update patches every bump.

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**